racket/src/cs/core/control.ss

```diff
@@ -172,6 +178,27 @@
             #t]
            [else (loop (cdr mc))])))))
 
+(define/who (maybe-block tag)
```

Comment:
"block" can mean many things. How about `maybe-future-barricade`, since "barricade" is more specific to the futures design?